lib/react.ml

```diff
@@ -1,4 +1,13 @@
 include Core
-module Dom = Dom
+
+module Dom = struct
+  include Dom
+
+  module Dsl = struct
+    module Html = Dom_html
+    module Svg = Dom_svg
+  end
```

Comment:
There's also the option to not include `Dom_html.Props` in `Dom_html`. Then everything is exported only once, which makes the docs easier to understand I think, and if you open them both you can disambiguate with `Html.div`, rather than having to use either `Html.Elements.div` or `Elements.div`, and `Html.Props.className` or just `Props.className`. The only downside is that you also have to `open Props` explicitly.

There's quite a bit of subtlety in these things though, and I'm sure we'll iron it out eventually.

Reply:
So consensus is to just remove `include Prop` from `Dom_html` then?

Reply:
Oops, sorry I merged without solving this. For now we can keep using existing solution and fine tune over time, I don't think it's a big deal in any case.